src/nvim/fileio.c

```diff
@@ -1992,13 +1992,15 @@ readfile (
 }
 
 #ifdef OPEN_CHR_FILES
-/*
- * Returns TRUE if the file name argument is of the form "/dev/fd/\d\+",
- * which is the name of files used for process substitution output by
- * some shells on some operating systems, e.g., bash on SunOS.
- * Do not accept "/dev/fd/[012]", opening these may hang Vim.
- */
-static int is_dev_fd_file(char_u *fname)
+/// Returns true if the file name argument is of the form "/dev/fd/\d\+",
+/// which is the name of files used for process substitution output by
+/// some shells on some operating systems, e.g., bash on SunOS.
+/// Do not accept "/dev/fd/[012]", opening these may hang Vim.
+///
+/// @param fname The file name to check
+///
+/// @return true if the file name argument is of the form "/dev/fd/\d\+",
+static bool is_dev_fd_file(char_u *fname)
```

Comment:
@cacplate AFAIK the only way to take others’ PR is make somebody with the necessary permissions close them and make your PR (not necessary in this order). Changes can be pulled from `refs/pull/$nr/head`: e.g. I have an alias `fpr = "!sh -c 'nr=\"$1\" ; shift ; git fetch upstream refs/pull/$nr/head:pr-$nr \"$@\"' -"` that when `git fpr 42` is run it creates local branch `pr-42` out of PR number 42.